test/linter.spec.js

```diff
@@ -0,0 +1,30 @@
+const utils = require('./helpers/utils'),
+  { mockCommand } = require('./helpers/mocks'),
+  { runCommand } = require('../generators/app/command'),
+  { exampleProjects } = require('./helpers/constants'),
+  commands = utils.getCommands();
+
+describe.each(exampleProjects)('Example project with %s', (projectName, { kickoffOptions }) => {
+  beforeAll(() => {
+    mockCommand();
+    return utils.runKickoff({
+      ...kickoffOptions,
+      nodeVersion: '8.9.12',
+      npmVersion: '6.4.1',
+      inTraining: false,
+      projectName: 'Project',
+      projectDescription: 'Project',
+      urlRepository: 'https://test.com.ar'
+    });
+  });
+
+  test.each(commands)('validate that there are no errors when generating the training files', command => {
```

Comment:
You need to check something after each command  or after all commands ? Also description of test it's ok ?

Reply:
1. you need to verify the commands to install depending on the CI
2. the description of the test was changed to "run the EsLinter for each project generated"

Reply:
Comments are okay but this comment still applies, do you need a test for each command (actual behaviour) or you need to check after all commands applies if any error is present (expected behaviour for me) ?